index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link href="styles.css" rel="stylesheet" type="text/css" />
+   <title>Mifrah Portfolio</title> 
+   <!-- Bootstrap CSS -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+
+<!-- Bootstrap Font Icon CSS -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css" />
+<link href="./styles.css" rel="stylesheet">
+</head>
+
+
+<body>
+   <div class="container">
+   <h1>MIFRAH'S ENGINEERING PORTFOLIO </h1>
+   <br>
+   <br>
+   <br> 
```

Comment:
Instead of using `<br>` to adjust spacing, I recommend using css `margin` or `padding`. In this case, you could add `margin-bottom` to `#container h1`.